i6_models/parts/conformer/norm.py

```diff
@@ -0,0 +1,28 @@
+import torch
+import torch.nn as nn
+
+
+class LayerNorm(nn.Module):
+    """
+    Generic LayerNorm that accepts any input shape
+    see here: https://pytorch.org/docs/stable/generated/torch.nn.LayerNorm.html
```

Comment:
I don't exactly understand what you mean.

I thought this layer norm was just to accept the (N, C, *) dim order, instead of (N, *, C) dim order. So it is not really about accepting any shape, but just this specific different dim order.

Reply:
ok then i made it very generic. i did not know that we restrict it to 3 tensor shape. anw this is simpler for me. i will just do this.

Reply:
I'm not saying to restrict it to 3 dims. I just say, this is about the dim order, not the shape.

Reply:
ok i think i misunderstood it. i will fix this.

Reply:
`LayerNormNC` or `LayerNormBC` or `LayerNormBatchFeatMajor` or so. Or simply keep `LayerNorm` and just explain it in the doc.

Btw, ESPnet has exactly this [here](https://espnet.github.io/espnet/_modules/espnet/nets/pytorch_backend/transformer/layer_norm.html).

Reply:
ah i see. should we follow then the implementation from ESPNet?

Reply:
I made it simpler.

Reply:
Yep looks good.